__tests__/endpoints.test.js

```diff
@@ -0,0 +1,28 @@
+const db = require("../db/connection");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+const request = require("supertest");
+const topics = require("../db/data/test-data/topics");
```

Comment:
This is not needed. We either get data back from the endpoint or use assertions/matchers to check this returned data is correct. Put another way data is only used to seed the database 👍

Reply:
I commented out these requires and the test failed?